built_w_pass.md

```diff
@@ -2,55 +2,54 @@
 
 Here is a list of projects we build successfully (Projects with a link were manually built)
 
-1. [ffmpeg](./oss-fuzz/projects/ffmpeg/build_w_pass.sh)
-2. [llvm](./oss-fuzz/projects/llvm/build_w_pass.sh)
-3. [json-c](./oss-fuzz/projects/json-c/build_w_pass.sh)
-4. [libtiff](./oss-fuzz/projects/libtiff/build_w_pass.sh)
-5. [lua](./oss-fuzz/projects/lua/build_w_pass.sh)
-6. alembic (auto_build_w_pass c++)
-7. [libpng](./oss-fuzz/projects/libpng/build_w_pass.sh)
-8. openssh (auto_build_w_pass c++)
-9. mongoose (auto_build_w_pass c++)
-10. libssh (auto_build_w_pass c++)
-11. json (auto_build_w_pass c++)
-12. cjson (auto_build_w_pass c++)
-13. capstone (auto_build_w_pass c++)
-14. boost (auto_build_w_pass c++)
-15. tmux (auto_build_w_pass c++)
-16. zlib (auto_build_w_pass c++)
-17. uriparser (auto_build_w_pass c++)
-18. draco (auto_build_w_pass c++)
-19. dropbear (auto_build_w_pass c++)
-20. example (auto_build_w_pass c++)
-21. exprtk (auto_build_w_pass c++)
-22. fmt (auto_build_w_pass c++)
-23. freeimage (auto_build_w_pass c++)
-24. fuzzing-puzzle (auto_build_w_pass c++)
-25. gfwx (auto_build_w_pass c++)
-26. gpac (auto_build_w_pass c++)
-27. clib (auto_build_w_pass c++)
-28. coturn (auto_build_w_pass c++)
-29. croaring (auto_build_w_pass c++)
-30. h3 (auto_build_w_pass c++)
-31. ibmswtpm2 (auto_build_w_pass c++)
-32. inchi (auto_build_w_pass c++)
-33. immer (auto_build_w_pass c++)
-34. libdwarf (auto_build_w_pass c++)
-35. libpg_query (auto_build_w_pass c++)
-36. libyang (auto_build_w_pass c++)
-37. llhttp (auto_build_w_pass c++)
-38. md4c (auto_build_w_pass c++)
-39. miniz (auto_build_w_pass c++)
-40. opensips (auto_build_w_pass c++)
-41. p11-kit (auto_build_w_pass c++)
-42. pycryptodome (auto_build_w_pass c++)
-43. grok (auto_build_w_pass c++)
-44. guezli (auto_build_w_pass c++)
-45. hoextdown (auto_build_w_pass c++)
-46. hostap (auto_build_w_pass c++)
-47. htslib (auto_build_w_pass c++)
-48. http-pattern-matcher (auto_build_w_pass c++)
-49. quickjs (auto_build_w_pass c++)
-50. utf8proc (auto_build_w_pass c++)
-51. w3m (auto_build_w_pass c++)
-52. zydis (auto_build_w_pass c++)
+1. [ffmpeg](./oss-fuzz/projects/ffmpeg/build_w_pass.sh),c++
+2. [llvm](./oss-fuzz/projects/llvm/build_w_pass.sh),c++
+3. [json-c](./oss-fuzz/projects/json-c/build_w_pass.sh),c++
+4. [libtiff](./oss-fuzz/projects/libtiff/build_w_pass.sh),c++
+5. [lua](./oss-fuzz/projects/lua/build_w_pass.sh),c
+6. alembic,c++ (auto_build_w_pass c++)
```

Comment:
please use the same line-format for autobuilds and the others so the parsing is easier

like
```md
5. [lua](./oss-fuzz/projects/lua/build_w_pass.sh),c
6. alembic(auto_build_w_pass c++),c++ 
```